### PR TITLE
feat(Screen of department-leader): Viewing ads and interest in them

### PR DESCRIPTION
**Pull request by JetoPistola**
Hi 😀 

I split the logic into 2 screens:

1. Screen of employer (Ramad)
2. Screen of candidate (Looking for job)

Screen Shot 2020-04-21 at 18 40 13

TODO: Add quick imports so there won't be `../../../../`....

### CODE REVIEW

File: src/types/Advertisements.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { Role } from "./Role";
 
 export interface Advertisement  {
     id: number;
-    name: String;
+    name: string;
```
Comment (JetoPistola) [inline src/types/Advertisements.ts:9]: This solved a bug, where typescript of material ui, can't use `String` elements, but expects `string`

Comment (SteveVaknin) [reply src/types/Advertisements.ts:9]: Nice

File: src/App.tsx [hunk 2]
```diff
@@ -3,14 +3,18 @@ import * as React from 'react';
...
 
 import JobsAppBar from './components/Application-Header/JobsAppBar';
-import JobsAppBody from './components/Application-Body/JobsAppBody';
+import JobsAppBody from './components/Home-Page/JobsAppBody';
 import jobsStore from './store/JobsStore';
+import JobsAppEmployerBody from './components/Personal-Zone/Employer/JobsAppEmployerBody';
 
 const App: React.FC<{}> = (): JSX.Element => {
+  // Question, can both Publisher and Contact, see the posts?
+  const isEmployer: boolean = true; // TODO: Fix this with logic from DB + jobsStore who is connected
```
Comment (SteveVaknin) [inline src/App.tsx:12]: userStore *

File: src/App.tsx [hunk 3]
```diff
@@ -3,14 +3,18 @@ import * as React from 'react';
 import { Provider } from 'mobx-react';
 
 import JobsAppBar from './components/Application-Header/JobsAppBar';
-import JobsAppBody from './components/Application-Body/JobsAppBody';
+import JobsAppBody from './components/Home-Page/JobsAppBody';
 import jobsStore from './store/JobsStore';
+import JobsAppEmployerBody from './components/Personal-Zone/Employer/JobsAppEmployerBody';
 
 const App: React.FC<{}> = (): JSX.Element => {
+  // Question, can both Publisher and Contact, see the posts?
```
Comment (SteveVaknin) [inline src/App.tsx:11]: I think only publisher for now

File: src/components/Personal-Zone/Employer/JobsAppEmployerBody.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import { useState, useEffect } from 'react';
+
+import { observer, inject } from 'mobx-react';
+
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+import { JobsStore } from '../../../store/JobsStore';
+import JobsList from './JobsList/jobsList';
+import Header from './header/Header';
+// import { Advertisement } from '../../types/Advertisements';
```
Comment (SteveVaknin) [inline src/components/Personal-Zone/Employer/JobsAppEmployerBody.tsx:11]: Remove comment

File: src/components/Personal-Zone/Employer/JobsList/job.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                            052-2222222
+                        </Typography>
+                        <Button className={classes.viewJobBtn} startIcon={<VisibilityOutlinedIcon className={classes.btnIcon} />}>
+                            <Typography variant='h6'>
+                                לצפייה ברזומה
+                            </Typography>
+                        </Button>
+                    </div>
+                    <div className={classes.candidate}>
+                        <Typography className={classes.candidateName} variant='h6' title='אאאאאבבבבבגגגגגדדדדד הההההווווו'>
```
Comment (SteveVaknin) [inline src/components/Personal-Zone/Employer/JobsList/job.tsx:74]: Very nice 🤣

File: src/App.tsx (hunk 2 above)
Comment (JetoPistola) [reply src/App.tsx]: Done!

**Pull request merged**